gensim/models/neighbor_sim.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from doc2vec import Doc2Vec
+from word2vec import Word2Vec
+
+from annoy import AnnoyIndex
+
+"""
+Using 3rd party k-NN libraries to improve performance for method DocvecsArray.most_similar().
+
+More information can be found at: https://github.com/piskvorky/gensim/wiki/Ideas-&-Feature-proposals#integrate-a-fast-k-nn-library
+
+Implemented by: Anh Le (anhldbk@gmail.com)
+
+Website: http://bigsonata.com
+
+Release 1: November 2015
```

Comment:
Again, belongs either to the file header, or -- even better -- delete completely (history changes are handled by git).